sdk/spring/azure-spring-cloud-stream-binder-servicebus-queue/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This release is compatible with Spring Boot 2.5.0 - 2.5.3 and Spring Cloud 2020.
 ### Dependency Upgrades
 - Upgrade to [spring-boot-dependencies:2.5.3](https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.5.3/spring-boot-dependencies-2.5.3.pom).
 
+### Breaking Changes
```

Comment:
We need to add the changelog content to azure-spring-integration-servicebus, azure-spring-cloud-starter-servicebus and azure-spring-cloud-autoconfigure